src/kvraft/tester.rs

```diff
@@ -211,6 +216,29 @@ impl Tester {
     }
 }
 
+lazy_static! {
+    static ref T0: Instant = Instant::now();
+}
```

Comment:
This looks weird, although I understand it's a workaround for getting current timestamp.
In madsim v0.2 we could use `std::time::SystemTime` directly to get timestamp of the simulated time.

Reply:
Seems there's a lot of work upgrading madsim...🥲 Now I removed the `lazy_static` stuff and used `t0` in `Tester` as a start timestamp.